src/FunderMaps.Core/Entities/Address.cs

```diff
@@ -59,6 +59,24 @@ public Address()
         [Required]
         public ExternalDataSource ExternalSource { get; set; }
 
+        /// <summary>
+        ///     External data source.
+        /// </summary>
+        [Required]
+        public string City { get; set; }
+
+        /// <summary>
+        ///     Building identifier.
+        /// </summary>
+        [Geocoder]
+        public string BuildingId { get; set; }
+
+        // TODO: Obsolete
+        /// <summary>
+        ///     Building identifier.
+        /// </summary>
+        public Building BuildingNavigation { get; set; }
```

Comment:
Do we want an entity within an entity here?

Reply:
No we don't, but there is a better solution proposed for a later time